src/sentry/models/exporteddata.py

```diff
@@ -57,18 +57,20 @@ def status(self):
         else:
             return ExportStatus.Valid
 
-    @property
-    def date_expired_string(self):
-        if self.date_expired is None:
-            return None
-        return self.date_expired.strftime("%-I:%M %p on %B %d, %Y (%Z)")
-
     @property
     def payload(self):
         payload = self.query_info.copy()
         payload["export_type"] = ExportQueryType.as_str(self.query_type)
         return payload
 
+    def get_date_string(self, date_field):
+        if not hasattr(self, date_field):
+            raise ValueError(u"Unrecognized field {} on ExportedData object".format(date_field))
+        if getattr(self, date_field) is None:
```

Comment:
@leeandher Nit: if you are going to use `hasattr` first to check if the property exists, might as well just use `self.date_field` instead of `getattr`.

Reply:
The `date_field` being supplied is just a string.
```python
```

Reply:
I thought about passing in the actual value, but then it got a little ugly:
```python
data_export.get_date_string(data_export.date_added)
```
and without writing three separate functions, I think this is probably a clean alternative

Reply:
I see, makes sense